creator/ingest_runs/signals.py

```diff
@@ -13,3 +16,41 @@ def ingest_run_pre_save(sender, instance, *args, **kwargs):
     if instance.versions.count() > 0:
         instance.name = instance.compute_name()
         instance.input_hash = instance.compute_input_hash()
+
+
+@receiver(post_transition, sender=IngestRun)
+def ingest_run_post_transition(sender, instance, name, source, target,
+                               *args, **kwargs):
+    """
+    Create a corresponding Event after the status of an IngestRun is changed.
+    """
+    if target == "running":
+        target = "started"
```

Comment:
Do you need to mutate the target here? Could you just map `running` onto `"IR_STA"` instead?

Reply:
It's for the Event messages. Maybe we can change the values of `state`?